motr/st/utils/copy.c

```diff
@@ -116,21 +116,19 @@ int main(int argc, char **argv)
 		      cp_param.cup_update_mode);
 	if (rc < 0) {
 		if (rc == -EEXIST) {
-			fprintf(stderr, "Object id: " U128X_F "exists. "
-					"To update an existing object, "
-					"use -u=start index! rc = %d\n",
-					 U128_P(&cp_param.cup_id),
-					 rc);
+			fprintf(stderr, "Object "U128X_F" already exists: "
+				"rc=%d. To update an existing object, use"
```

Comment:
I myself prefer to old code.

String on this line, is continuous to the one on the previous line.
So, they are aligned.

Reply:
Ready, please check.